src/battle_message.c

```diff
@@ -1925,10 +1929,10 @@ const u16 gWeatherStartsStringIds[] =
     [WEATHER_RAIN]               = STRINGID_ITISRAINING,
     [WEATHER_SNOW]               = (B_OVERWORLD_SNOW >= GEN_9 ? STRINGID_STARTEDSNOW : STRINGID_STARTEDHAIL),
     [WEATHER_RAIN_THUNDERSTORM]  = STRINGID_ITISRAINING,
-    [WEATHER_FOG_HORIZONTAL]     = STRINGID_ITISRAINING,
+    [WEATHER_FOG_HORIZONTAL]     = (B_OVERWORLD_FOG == GEN_4 ? STRINGID_FOGISDEEP : STRINGID_ITISRAINING),
```

Comment:
I don't think we need the config check here considering the "It is raining" string is never actually used and just seems to be a placeholder.